### PR TITLE
fix: remove old nodes

### PR DESCRIPTION
**Pull request by stepanLav**
Change nodes, based on test results:
https://novasamatech.github.io/test-runner/5573/

### CODE REVIEW

File: chains/v20/chains.json [hunk 1]
```diff
@@ -4562,8 +4562,8 @@
         ],
         "nodes": [
             {
-                "url": "wss://fullnode.parachain.centrifuge.io",
```
Comment (stepanLav) [inline chains/v20/chains.json:4565]: that was a node for Centrifuge EVM

File: chains/v20/chains.json [hunk 2]
```diff
@@ -5992,7 +5992,7 @@
                 "name": "TraceLabs node"
             },
             {
-                "url": "wss://origintrail-rpc.dwellir.com",
```
Comment (stepanLav) [inline chains/v20/chains.json:5995]: dwellir changed node name

File: chains/v20/chains.json [hunk 3]
```diff
@@ -7707,10 +7707,6 @@
             {
                 "url": "wss://darwiniacrab-rpc.dwellir.com",
                 "name": "Dwellir node"
-            },
-            {
-                "url": "wss://crab-rpc.darwiniacommunitydao.xyz",
```
Comment (stepanLav) [inline chains/v20/chains.json:7712]: no longer available

File: chains/v20/chains.json (hunk 1 above)
Comment (leohar) [reply chains/v20/chains.json:4565]: I would prefer not remove this node from chains.json as extracting EVM chains from Astar, Centrifuge and other is in development stage atm

**Review by leohar — APPROVED** [review]

**Pull request merged**